src/main/java/org/example/springintro/dto/CreateBookRequestDto.java

```diff
@@ -9,7 +11,10 @@ public class CreateBookRequestDto {
     private String title;
     private String author;
     private String isbn;
+    @Min(0)
```

Comment:
```suggestion
    @Min(0)
    @NotNull
```

where is validation for other properties?